workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSReceiveSystem.cs

```diff
@@ -154,6 +170,158 @@ private void OnCommandResponse(CommandResponseOp op)
             specificDispatcher.OnCommandResponse(op);
         }
 
+        private void OnCreateEntityResponse(CreateEntityResponseOp op)
+        {
+            if (!createEntityStorage.CommandRequestsInFlight.TryGetValue(op.RequestId.Id, out var requestBundle))
+            {
+                throw new InvalidOperationException($"Could not find corresponding request for RequestId {op.RequestId.Id} and command CreateEntity.");
```

Comment:
Possibly should make this an error for the logger, as there is nowhere for this to be caught and it doesn't put us in an unrecoverable state.